packages/swing-store/src/bundleStore.js

```diff
@@ -0,0 +1,242 @@
+// @ts-check
+import { createHash } from 'crypto';
+import { Readable } from 'stream';
+import { Buffer } from 'buffer';
+import { encodeBase64, decodeBase64 } from '@endo/base64';
+import { checkBundle } from '@endo/check-bundle/lite.js';
+import { Fail, q } from '@agoric/assert';
+import { buffer } from './util.js';
+
+/**
+ * @typedef { { moduleFormat: 'getExport', source: string, sourceMap: string? } } GetExportBundle
```

Comment:
Our `bundleStore` will only accept `EndoZipBase64` -format bundles, so it's probably best to remove these other two typedefs, and define the API in terms of `EndoZipBase64Bundle` instead of `Bundle_proper`, so folks will get a TSC error if they don't check the bundle type before attempting to pass it to `bundleStore`

If/when we expand it to other kinds of bundles, we can expand the types.

Reply:
actually don't remove the other two: the patch I sent you replaces the definition of `Bundle` with the three-flavor union, and I'd like to keep that definition because my upcoming `b0-` / nestedEvaluate PR will use it.